source/MapPlanetCard.cpp

```diff
@@ -44,10 +44,27 @@ MapPlanetCard::MapPlanetCard(const StellarObject &object, unsigned number, bool
 	hasShipyard = planet->HasShipyard();
 	hasOutfitter = planet->HasOutfitter();
 
-	reputationLabel = !hasSpaceport ? "No Spaceport" :
-		GameData::GetPolitics().HasDominated(planet) ? "Dominated" :
-		planet->GetGovernment()->IsEnemy() ? "Hostile" :
-		planet->CanLand() ? "Friendly" : "Restricted";
+	if(!hasSpaceport)
+		reputationLabel = "No Spaceport";
+	else
+	{
+		switch(planet->GetFriendliness())
+		{
+			default:
```

Comment:
I would remove it tbh, because some compilers warn if an enum switch doesn't have a case for every value.